docs: update homepage settings (#75154)
metabase/metabase: docs/configuring-metabase/settings.md

> Note: if you haven't set up HTTPS on your server, Metabase will not let you enable HTTPS redirect. Instead, you'll get a warning saying "It looks like HTTPS is not properly configured."

## Homepage

> Set what people see when they land on your Metabase. Choose between:

Reviewer: ```suggestion
Set the page people see when they first log in to your Metabase. Choose between:
```

- **Default Metabase home**: The standard Metabase homepage.
- **Dashboard**: Pick a dashboard to serve as the homepage. If people lack permissions to view the selected dashboard, Metabase will redirect them to the default homepage.
- **Custom URL**: Send people to a specific URL (a collection, question, dashboard, or other page). Make sure people have access to the URL.

> > The **Custom URL** option is only available on Pro and Enterprise plans with whitelabeling enabled.

Reviewer: ```suggestion
- **Custom URL**: Send people to a specific URL (a collection, question, dashboard, or other page). Make sure people have access to the URL. This option is only available on Pro and Enterprise plans with whitelabeling enabled.
```

> > The **Custom URL** option is only available on Pro and Enterprise plans with whitelabeling enabled.

Reviewer 2: What does "with whitelabeling enabled" means? How would someone enable whitelabeling?
